pkgs/development/python-modules/pep517/default.nix

```diff
@@ -41,9 +42,18 @@ buildPythonPackage rec {
 
   preCheck = ''
     rm pytest.ini # wants flake8
-    rm tests/test_meta.py # wants to run pip
   '';
 
+  disabledTests = [
+    # these import setuptools, which for some reason cannot be found
```

Comment:
you could add setuptools to checkInputs

Reply:
That didn't seem to prevent hitting the network to download `setuptools`. I should clarify the comment to indicate that it's not strictly about importing `setuptools`.